src/aof.c

```diff
@@ -1608,7 +1608,7 @@ int rewriteAppendOnlyFile(char *filename) {
         /* Update COW info */
         long long now = mstime();
         if (now - cow_updated_time >= 1000) {
-            sendChildInfo(CHILD_INFO_TYPE_CURRENT_INFO, 0, "AOF rewrite");
+            sendChildInfo(CHILD_INFO_TYPE_CURRENT_INFO, dbTotalServerKeyCount(), "AOF rewrite");
```

Comment:
i think we better cache this before the `while (bytes_to_write) {` loop.
this function has a loop inside it.